scripts/updategroupids.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+  
+MONGO_URI=$1
+DBNAME=$2
+
+
+while IFS=, read -r eventid platformid title createby startdate enddate groupname groupid
+do
+    # echo "$eventid, $platformid, $groupid"
+    event=$(mongo $MONGO_URI --quiet --eval "db.$DBNAME.find({_id: ObjectId('"$eventid"')})")
```

Comment:
I think the collection name is also needed here.

Reply:
yup, actually that should be collection name.